web/portal/brand/src/entities/Administrator/Administrator.tsx

```diff
@@ -26,6 +26,7 @@ const properties: AdministratorProperties = {
   email: {
     label: _('Email'),
     maxLength: 100,
+    required: false,
```

Comment:
Does this work? I mean, it already was not required

Reply:
But it marks as required in Brand portal.

![image](https://user-images.githubusercontent.com/101095813/221522089-30335c14-9b0a-453f-9ea7-515c641c6687.png)